.github/docker-compose.test.yml

```diff
@@ -14,11 +14,27 @@
 services:
   # The Unleash server contains the Unleash configuration and
   # communicates with server-side SDKs and the Unleash Proxy
+  frontend:
+    image: node:20-alpine
+    working_dir: /app
+    volumes:
+      - ..:/app
+    command: |
+      sh -c "echo \"Building frontend\" && \
+      yarn --cwd ./frontend install && \
+      echo \"Waiting for Unleash to start\" && \
+      npx wait-on http://unleash:4242/health && \
+      echo \"Unleash is up, starting frontend\" && \
+      yarn --cwd ./frontend run dev"
```

Comment:
This is what's needed to run the frontend. 

This also made me realize that our frontend depends on code from the server:
`Failed to resolve import "@server/util/escape-regex" from "src/component/common/Highlighter/Highlighter.tsx". Does the file exist?`
so we need to load the whole server into this docker image